configure.ac

```diff
@@ -66,14 +66,21 @@ AC_ARG_ENABLE([urpc],
     [enable_urpc=$enableval],
     [enable_urpc=no]
 )
-AC_ARG_ENABLE([perfflow],
-    [AS_HELP_STRING([--enable-perfflow],
+AM_CONDITIONAL([URPC], [test "x$enable_urpc" = "xyes"])
```

Comment:
The AM_CONDITIONAL is not matching with AC_ARG_WITH. Fix the order to keep them together.

Reply:
I'm not sure what you mean here. Unless GitHub did something dumb when I pushed, the `AM_CONDITIONAL` immediately follows the relevant `AC_ARG_<ENABLE|WITH>`

Reply:
Oh i get it. Its all one after the other. I think for better code readability add a new line with a comment saying adding UCX/URPC/PERFLOW OPTIONS.